### DISCUSSION TITLE
请问大佬们，更新2.0之后为什么A4K的所有着色器都用不了了？

### DISCUSSION THREAD

**Discussion by GitHubPi-YueXin** [diana7127/mpv.net-DW#27] [Q&A] [answered]
视频滤镜都能用，就看动漫的这个都失效了，1.8之前的版本都能用

**Comment by GitHubPi-YueXin** [comment]
而且好像也找不到看杜比视界的选项了

**Comment by diana7127** [comment, marked as answer]
不好意思，安装包中的shader文件夹丢失了，已重新打包，麻烦重新下载一下安装包哦；
mpv_lazy的2024V0版本中移除了` [HDR_DolbyVision]` ，需要的话可以将上个版本的` [HDR_DolbyVision]` 写入profiles.conf文件中

**Reply by diana7127** [reply]
在profiles.conf文件中写入以下信息
```editorconfig
[HDR-DolbyVision]
 profile-desc = hdr杜比视界
 profile-cond = filename:match("%.DoVi")~=nil
 profile-restore = copy
 vo = gpu-next
 hdr-compute-peak = no
 tone-mapping = st2094-40
```

**Reply by GitHubPi-YueXin** [reply]
太感谢您了！把这么优秀的作品无私奉献给我们！大佬，还有两个问题，请问能否同时挂载超分滤镜和插帧滤镜？会不会很吃电脑性能？还有一个就是能否在观看杜比视界视频的同时挂载插帧滤镜和拖动进度条？感谢大佬！

**Reply by diana7127** [reply]
超分滤镜非常吃电脑性能，可以打开控制台看看会不会卡；杜比视界的话还是不能拖动进度条

**Reply by GitHubPi-YueXin** [reply]
好的，非常感谢！提前祝您新年快乐！
